docs/how-to/store-large-files.md

```diff
@@ -29,3 +29,23 @@ A 1MiB sector is 1,048,576 bytes. Next, we multiply that by 254/256. Result: a 1
 | 8MiB        | 8MB (8,000,000 bytes)       | 8,323,072 bytes       |
 | 512MiB      | 512MB (512,000,000 bytes)   | 532,676,608 bytes     |
 | 32GiB       | 32GB (32,000,000,000 bytes) | 34,091,302,912 bytes  |
+
+### Storage deals with offline data transfer
+Filecoin's offline data transfer feature is recommended for petabyte-scale datasets and larger. This allows users with very large datasets to complete the data transfer step offline (e.g. by shipping hard drives from the client to the storage miner), while the storage deal continues to work as intended on-chain.
+
+It is implemented via a flag on the storage deal command that tells the client not to transfer the data over the network, and provides a piece CID (a unique identifier describing the data) to the miner instead, which a miner must then match for the deal to go through. This gives the client node flexibility in how it can set up the deal — for example, passing miners a specific location on a hard drive for the data they can use to generate the piece CID. 
+
+How to make storage deals with offline data transfer:
+1. Generate the unique pieceCID. (TODO include steps or links.)
+
+2. Identify a suitable miner. (TODO how?)
+
```

Comment:
I'm not sure how answerable this question is -- are we planning on providing some platforms for miners to connect with potential clients interested in offline transfers, and like, give them their physical address?

Reply:
could this not be a function that gets built directly into explorers etc? At least providing a data set on the nodes supplying their advertised IP / geolocation if such a thing has value. Not a perfect solution, but it's something.

Reply:
Not currently and not planned before mainnet, so I think we need to write a more general summary of the process and step back from tutorial-style.